addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -428,15 +428,18 @@
         </div>
 
         <div class="d-flex flex-row">
+            <a t-if="channel.can_upload" href="#">
+                <span t-att-data-slide-id="slide.id" t-attf-class="o_wslides_js_slide_toggle_is_preview badge #{'badge-success py-1' if slide.is_preview else 'badge-light badge-hide border'} font-weight-normal px-2 m-1">Free preview</span>
+            </a>
+            <span t-elif="slide.is_preview and not channel.is_member" class="badge badge-success font-weight-normal px-2 py-1 m-1">Free preview</span>
+            <span t-if="slide.is_new_slide and not channel_progress[slide.id].get('completed')" class="badge badge-info badge-arrow-right font-weight-normal px-2 py-1 m-1">
+                New
+            </span>
```

Comment:
Are you sure it's necessary to move this block ? I think you can only adapt without moving everything.

Reply:
No ok, you can keep it like this, for design bioutifulness reason